server/test/deviceController.test.ts

```diff
@@ -0,0 +1,49 @@
+import axios, { AxiosResponse } from 'axios';
+import { Device } from '../src/types/device';
+
+const mock: Device = expect.objectContaining({
+    id: expect.any(String),
+    model: expect.any(String),
+    name: expect.any(String),
+    automations: expect.any(Array),
+    nameByUser: expect.any(String),
+    areaId: expect.any(String),
+    disabledBy: expect.any(String),
+} as Device);
+
+const HaWeatherDeviceId = '45eb8f705b20fa1df358c7d22d8ffaf8';
+
+const baseUrl: string = 'http://localhost:3000';
+
+describe('Device controller test', () => {
+    test('it should get devices', async () => {
```

Comment:
maybe "it" in the test name is not necessary

Reply:
Maybe its not a big deal (it still makes sense right ?)

Reply:
Just a suggestion